### PR TITLE
The detail book component is created

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary
- Completed the detailed texts of the book

## Screenshot
book detail I

## Trello Card
https://trello.com/c/wnY9YFTD/26-detalle-de-libro-i-textos

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (eledwinn) [inline .gitignore]: Please add empty lines at end of all files

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```
Comment (eledwinn) [inline index.html:1]: Use two spaces for indent all files

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+				<li>
+					<h2>Año de publicación:</h2>
+					<span>Año de publicación</span>
+				</li>
+			</ul>
+		</div>
+	</div>
+
+</body>
+</html>
```
Comment (eledwinn) [inline index.html]: Empty line at EOF :P

File: scss/_colors.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+$black: #000;
+$gray: #828282;
```
Comment (eledwinn) [inline scss/_colors.scss]: Same

File: scss/detail.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.book {
+	color: $black;
+
+	&__title {
+		margin: 0 0 52px 0;
+		h1 {
```
Comment (eledwinn) [inline scss/detail.scss]: Avoid to use element name selector in your CSS, we try using classes. Check our Style Guide here https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md

File: scss/detail.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+		}
+
+		span {
+			font-size: 30px;
+			font-weight: bold;
+		}
+	}
+
+	&__detail ul li{
+		margin: 0 0 36px 0;
```
Comment (eledwinn) [inline scss/detail.scss]: Empty line between prop and selector please

File: scss/detail.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+		h2 {
+			font-size: 22px;
```
Comment (eledwinn) [inline scss/detail.scss]: If your `h2` and `span` tags have the same `font-size`, try set in same selector block.

**Review by eledwinn — CHANGES_REQUESTED** [review]
Great job men. 👏 👏 👏

File: scss/detail.scss [hunk 8]
```diff
@@ -0,0 +1,35 @@
+.book {
+  color: $black;
+
+  &__title {
+    margin: 0 0 52px 0;
```
Comment (CrisVega08) [inline scss/detail.scss]: margin-bottom: 52px;

File: scss/detail.scss [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+      font-weight: 700;
+    }
+
+    &__text--secundary {
+      font-size: 30px;
+      font-weight: bold;
+    }
+  }
+
+  &__detail ul li{
```
Comment (CrisVega08) [inline scss/detail.scss]: avoid tag selectors

File: scss/detail.scss [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    margin: 0 0 36px 0;
```
Comment (CrisVega08) [inline scss/detail.scss]: margin-bottom: 36px;

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 9 above)
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  &__detail ul li {
```

**Review by eledwinn — APPROVED** [review]
Cool!!! 😎

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**